modules/self_managed_nodes/win_cm_vpc_cni.tf

```
@@ -0,0 +1,22 @@
+locals {
+  aws_auth_configmap_data = {
+    enable-windows-ipam = "true"
+  }
+}
+
+resource "kubernetes_config_map" "amazon-vpc-cni" {
```

```suggestion
resource "kubernetes_config_map" "amazon_vpc_cni" {
```

Snake case for names.